src/calcium/CalciumTypes.jl

```diff
@@ -59,11 +59,31 @@ end
 #
 ################################################################################
 
+qqbar_field_doc = raw"""
+    QQBarField <: Field
+    QQBarFieldElem <: FieldElem
+
+The algebraic closure $\overline{\mathbb Q}$ of the rationals $\mathbb Q$ and its elements.
+For convenience, we predefine the global variable `const QQBar = QQBarField()`.
```

Comment:
I don't know if this confuses users, as this global does not exist in Oscar. Maybe it's better to advertise `algebraic_closure(QQ)` or write `Nemo.QQBar`.

Reply:
Do we have `algebraic_closure(QQ)` in Nemo?

Reply:
https://github.com/Nemocas/Nemo.jl/blob/928e06ae0fa0735fbca5da42b902ac15a6cedfa7/src/calcium/qqbar.jl#L1533

Reply:
Then I suggest we change the docstring as you suggested